cognitive_mapping_and_planning/tfcode/vision_baseline_lstm.py

```diff
@@ -196,6 +197,7 @@ def combine_setup(name, combine_type, embed_img, embed_goal, num_img_neuorons=No
       out = embed_goal
     else:
       logging.fatal('Undefined combine_type: %s', combine_type)
+      raise ValueError('Undefined combine_type: %s', combine_type)
```

Comment:
I think @martinwicke means that the proposed change won't have any effect as the program will crash at logging.fatal (and thus never get to raise ValueError). The intended behavior is to raise a ValueError so perhaps we should get rid of the logging.fatal.